src/main/kotlin/org/rust/ide/annotator/RustHighlightingAnnotator.kt

```diff
@@ -12,7 +12,11 @@ import org.rust.lang.core.psi.util.elementType
 
 // Highlighting logic here should be kept in sync with tags in RustColorSettingsPage
 class RustHighlightingAnnotator : Annotator {
-    override fun annotate(element: PsiElement, holder: AnnotationHolder) = element.accept(object : RustElementVisitor() {
```

Comment:
Nitpicking: using expression syntax here allowed us to reduce level of indentation.